Replace resources page

mosabua opened the pull request:
Much improved replacement for https://trino.io/resources.html :

- Remove hardcoded old page
- Add content to data file
- Add layouts and includes for data driven pages
- Add header navigation
- Add images
- Add wording and links as need

colebow commented on ecosystem/add-ons-intro.md:
```diff
@@ -0,0 +1,2 @@
+Add-ons provide support for additional use cases and scenarios for your
```
Maybe "provide additional utility and expand on what Trino has to offer" or something like that? "additional use cases and scenarios" doesn't really mean much.

colebow commented on ecosystem/clients-intro.md:
```diff
@@ -0,0 +1,6 @@
+Clients allow you to connect to Trino, submit SQL queries, and receive the
+results. Clients are full-featured applications or libraries and drivers that
+allow you to connect to your custom application. Clients simple command line
```
"Clients simple command line" isn't right.

colebow commented on ecosystem/solution.md:
```diff
@@ -0,0 +1,14 @@
+
+Solutions integrate Trino to create a full-featured product for specific use
+cases. They can simplify Trino management, provide additional features and
+extensions such as security and access control enhancement, or adapt Trino to
```
```suggestion
extensions, such as security and access control enhancement, or adapt Trino to
```

colebow commented on ecosystem/solutions-intro.md:
```diff
@@ -0,0 +1,4 @@
+Solutions integrate Trino to create a full-featured product for specific use
+cases. They can simplify Trino management, provide additional features and
+extensions such as security and access control enhancement, or adapt Trino to
```
```suggestion
extensions, such as security and access control enhancement, or adapt Trino to
```

We're duplicating this text in solution.md and solutions-intro.md. Is that correct?

mosabua replied:
Opps .. fixed

mosabua commented on _includes/toolslist.html:
```diff
@@ -0,0 +1,33 @@
+          {% endif %}
+        </div>
+        <div class="col-md-9 col-s-12">
+<div markdown="1">### {{tool.name}}
```
This weird markdown and html mixture is necessary to make the right hand nav work automatically

mosabua commented:
Demoed to @martint and addressed feedback. Also got feedback from @dain and addressed that. 

From our discussions on slack this is essentially approved now and I am going to merge in the interest of getting this huge improvement live. Also just to clarify we will follow this up with adding more clients and add-ons soon in separate PRs. And another PR will add a whole separate section for data sources that basically will refer to all the connector that are part of trinodb and beyond. 

This will be a great asset overall for showing the success of the ecosystem around the project and the momentum of support and collaboration with other communities.

nineinchnick commented:
You now can't search for text on this page and have to visually inspect all logos. Won't that affect SEO?
